Test/DotNet.Tests.ps1

```diff
@@ -206,3 +213,18 @@ Describe 'DotNet.when command is missing' {
         ThenTaskFailedWithError 'is\ required'
     }
 }
+
+Describe 'DotNet.when passing paths to the command and working directory isn''t the build root' {
+    AfterEach { Reset }
+    It 'should resolve paths' {
+        Init
+        GivenDotNetCoreProject 'DotNetCore\DotNetCore.csproj' -Targeting 'netcoreapp2.0'
+        GivenDotNetCoreProject 'DotNetCore\DotNetCore2.csproj' -Targeting 'netcoreapp2.0'
+        WhenRunningDotNet 'build' -WithPath 'DotNetCore\*.csproj' -WithSdkVersion '2.*' -InWorkingDirectory 'DotNetCore'
+        ThenProjectBuilt 'DotNetCore\bin\Debug\netcoreapp2.0\DotNetCore.dll'
+        ThenProjectBuilt 'DotNetCore\bin\Debug\netcoreapp2.0\DotNetCore2.dll'
+        ThenLogFile 'dotnet.build.DotNetCore.csproj.log' -Exists
+        ThenLogFile 'dotnet.build.DotNetCore2.csproj.log' -Exists

```

Comment:
Add a `ThenWarningMessage`? We want to make sure we're writing a warning to get users off the deprecated functionality.